src/Api/Service/Sf.php

```diff
@@ -413,20 +413,19 @@ public function projectcode_exists($code)
     // ---------------------------------------------------------------
     // Activity Log
     // ---------------------------------------------------------------
-    public function valid_activity_types_dto()
-    {
-        return ActivityListDto::getActivityTypes($this->website);
-    }
-
```

Comment:
this and the following two comments also have me re-thinking that "activity feed" button altogether... @megahirt do you think we should leave that button in the legacy app or hide it and just leave all these methods Robin is suggesting we not remove?

Reply:
This PR is actually about replacing the legacy activity feed with a new activity feed provided by LFNext.

1) The new activity feed is for the current project only.  A "project dashboard" of sorts.
2) The old activity feed can be removed from the legacy app.  No need to keep it around.
3) The PHP function valid_activity_types should be removed since it is not used in the new LFNext activity feed.

A few things currently missing from this PR:

1) clean up angularJS files no longer needed: angularJS services and templates related to activity feed
2) remove references to validActivityTypes

Reply:
@megahirt @rmunn please let me know you've done this so I can resolve this conversation.

Reply:
If we're removing the "Activity Feed" button entirely from the legacy app, then there is more code that can be removed:

* `activity_list_dto_for_lexical_entry` in Sf.php and RightsHelper.php (was used in the `listActivityForLexicalEntry` function in activity.service.ts, but activity.service.ts is entirely gone)
* `getActivityForOneLexEntry` and `getActivityForLexEntry` methods in ActivityListDto.php (only used by the `activity_list_dto_for_lexical_entry` code in Sf.php)
* `ActivityListModelByLexEntry` class in ActivityListDto.php (only used in the `getActivityForLexEntry` method which is being removed)

I actually like having a per-entry activity feed, so I don't quite agree with removing that button or the code that feeds it, since it might be a little while before the editor view gets converted to LFNext, and until that point, it's still useful to have activity-list code in the legacy app. So I agree with @longrunningprocess 's initial design of keeping the legacy-app "Activity Feed" button around. Let's have that conversation next time we're able to all meet together, and decide whether the activity feed should really be ripped out of the legacy app or not.

Reply:
I reverted that stuff today @rmunn , things should be back the way they were now.